Description: Fixes problem with ubertooth-dfu not found. The install script
 assumes that ubertooth-dfu is already in the path. It is not when building the
 deb package, but it is available from a relative path since it has already
 been built.
Author: Ruben Undheim <ruben.undheim@gmail.com>

Index: ubertooth/firmware/common.mk
===================================================================
--- ubertooth.orig/firmware/common.mk	2017-01-22 11:21:49.060809296 +0100
+++ ubertooth/firmware/common.mk	2017-01-22 11:22:28.523474935 +0100
@@ -259,7 +259,8 @@
 REMOVE = rm -f
 
 # DFU tool should be ubertooth-dfu
-DFU_TOOL ?= $(strip $(shell which ubertooth-dfu))
+#DFU_TOOL ?= $(strip $(shell which ubertooth-dfu))
+DFU_TOOL = ../../obj-$(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)/ubertooth-tools/src/ubertooth-dfu
 
 # Define Messages
 # English
